### DISCUSSION TITLE
点击节点移动过程中，碰到其他节点，怎么停止移动

### DISCUSSION THREAD

**Discussion by avrinfly** [antvis/X6#1859] [Q&A]
画布中，两个节点，点击移动一个节点，如何才能不碰到另一个节点，且另一个节点不移动位置

两个节点没有从属关系，是相互独立的

**Comment by NewByVector** [comment]
可以使用 [restrict](https://x6.antv.vision/zh/docs/tutorial/basic/interacting/#restrict) 限制节点移动范围

**Reply by avrinfly** [reply]
> 可以使用[限制](https://x6.antv.vision/zh/docs/tutorial/basic/interacting/#restrict)限制节点移动范围

这个节点移动范围我理解是，节点只能在这个范围内移动，我想实现的效果是在除了这个区域的其他地方可以移动，这个范围内不移动。
有方法能做到吗

**Reply by NewByVector** [reply]
理解错你的意思了，现在还不支持的。
